src/main/java/seedu/duke/data/notebook/Notebook.java

```diff
@@ -50,15 +186,15 @@ public void addNote(Note note) {
     }
 
     /**
-     * Removes a note from the timetable.
+     * Removes a note from the notebook.
      */
     public boolean deleteNote(int index) {
```

Comment:
No need for this 1 line function.

Reply:
This function isn't for me/nor did I create it. Used in DeleteNoteCommand